src/dmd/target.d

```diff
@@ -399,22 +387,24 @@ extern (C++) struct Target
      */
     extern (C++) uint alignsize(Type type)
     {
+        import dmd.mtype : TY = ENUMTY;
```

Comment:
This import appears 5 times in this PR. I don't know what your threshold is, but it looks like a good candidate for the struct level imports.

Reply:
Also, if we go down this road, does this mean we can get rid of all the T* aliases in dmd.mtype ?

Reply:
Someone tried removing `ENUMTY` before, can give it a retry. #12579